demos/using-dev-build/dfx.json

```diff
@@ -33,8 +31,7 @@
       "__4": "(which in this case populates 'dist/', see package.json).",
       "__5": "source must be a directory, otherwise we get a cryptic error.",
       "source": ["dist/"],
-
-      "__6": "NOTE: the 'frontend' value is needed otherwise HTTP calls are not proxied; value can be anything though.",
+      "__6": "NOTE: the 'frontend' signals dfx that it is useful to print out the front-end URL of this canister when deploying. It is not strictly required.",
```

Comment:
does `dfx` now proxy to the canister by default?

Reply:
Yes.